backend/src/utils/validation/isValidCourseData.ts

```diff
@@ -0,0 +1,19 @@
+import { ICreateCourseBody } from 'interfaces/ICourses/IQueryCourses';
+import isValidText from './isValidText';
+import isValidComplexity from './isValidComplexity';
+import isValidMaterials from './isValidMaterials';
+import isValidQuestions from './isValidQuestions';
+import isValidAvatar from './isValidAvatar';
+
+const isValidCourseData = (courseData: ICreateCourseBody): boolean => {
+  return (
+    isValidText(courseData.title) &&
+    isValidComplexity(courseData.complexity) &&
+    isValidText(courseData.description) &&
+    isValidMaterials(courseData.materials) &&
+    isValidQuestions(courseData.test?.questions) &&
+    isValidAvatar(courseData.avatar)
+  );
+};
```

Comment:
That's nor really good. Pls, call every validation separately and throw an error, that show what actually was wrong. Example: if description is invalid, the message should be "Course description was not validated successfully." or "Invalid course description."